Aquilaweb.Support.PendingReboot/details.json

```diff
@@ -0,0 +1,12 @@
+{
+  "ManagementPackSystemName": "Aquilaweb.Support.PendingReboot",
+  "ManagementPackDisplayName": "Aquilaweb Support - Pending Reboots,

```

Comment:
There is a missing quote at the end of this line

(I know this is merged already but this issue still exists in master: https://github.com/squaredup/scom-community-catalog/blob/master/Aquilaweb.Support.PendingReboot/details.json and the open PR as well)